integrations.json

```diff
@@ -332,8 +332,8 @@
           "assembly": "MongoDB.Driver.Core",
           "type": "MongoDB.Driver.Core.WireProtocol.IWireProtocol",
           "method": "Execute",
-          "minimum_major": 0,
-          "minimum_minor": 0,
+          "minimum_major": 2,
+          "minimum_minor": 2,
```

Comment:
For the original integration, don't we want to set the maximum version to 2.65535 so it only applies to 2.2 - 2.x, but not for 3.x?